recipes/gromacs_py/meta.yaml

```diff
@@ -28,6 +28,7 @@ requirements:
     - pdb_manip_py
     - pdb2pqr_htmd_propka30
     - gromacs <2020
```

Comment:
@samuelmurail do you support the most recent version of gromacs yet?

Reply:
Yes it does, however, there is a bug in gromacs 2020 with C-ter capping (see bug report https://redmine.gromacs.org/issues/3301).
This make the gromacs_py test failing, however for a basic use of gromacs there is no issue.

Reply:
Okay, then I guess this can be merged, unless you want to update the gromacs pin

Reply:
Ready to merge.
Thanks for the review.

Cheers

Reply:
You're welcome.

Just out of curiosity, what is the benefit of this package, compared to the built-in python API? I never used it, but I understood it was introduced in version 2020.

Reply:
Well, first reason is that I started the gromacs_py package three years ago, before the official API and since I kept working on it.
Anyway the main goal is also different, my package intends to simplify MD simulation to non experts, also it allows to modify molecules topologies and coordinates (eg. peptide cyclisation).